libs/telemetry/src/main/java/org/opensearch/telemetry/tracing/Context.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing;
+
+import org.opensearch.telemetry.tracing.attributes.Attributes;
+
+/**
+ * Context for span details.
+ */
+public final class Context {
```

Comment:
May be 

```suggestion
public final class SpanCreationContext {
```

The `Context` is heavily used in Otel, it will be super confusing to deal with those two

Reply:
Thanks for suggesting the name, I was totally running out of the ideas :)